crates/router/src/core/payments/operations/payment_capture.rs

```diff
@@ -225,3 +259,58 @@ impl<F: Send + Clone> ValidateRequest<F, api::PaymentsCaptureRequest> for Paymen
         ))
     }
 }
+
+impl PaymentCapture {
+    async fn create_capture_and_update_attempt(
+        state: &AppState,
+        authorized_payment_attempt: storage::PaymentAttempt,
+        storage_scheme: enums::MerchantStorageScheme,
+    ) -> CustomResult<(storage::Capture, storage::PaymentAttempt), errors::StorageError> {
+        state
+            .store
+            .insert_capture(
+                Self::make_capture(&authorized_payment_attempt),
+                authorized_payment_attempt,
+                storage_scheme,
+            )
+            .await
+    }
+    fn make_capture(authorized_attempt: &storage::PaymentAttempt) -> storage::CaptureNew {
+        let capture_sequence = authorized_attempt.multiple_capture_count.unwrap_or(0) + 1;
+        let now = common_utils::date_time::now();
+        storage::CaptureNew {
+            payment_id: authorized_attempt.payment_id.clone(),
+            merchant_id: authorized_attempt.merchant_id.clone(),
+            capture_id: format!("{}_{}", authorized_attempt.attempt_id, capture_sequence),
+            status: enums::CaptureStatus::Started,
+            amount: authorized_attempt
+                .amount_to_capture
+                .unwrap_or(authorized_attempt.amount),
+            currency: authorized_attempt.currency,
+            connector: authorized_attempt.connector.clone(),
+            error_message: None,
+            tax_amount: None,
+            created_at: now,
+            modified_at: now,
+            error_code: None,
+            error_reason: None,
+            authorized_attempt_id: authorized_attempt.attempt_id.clone(),
+            capture_sequence,
+            connector_transaction_id: None,
+        }
+    }
+    #[instrument(skip_all)]
+    pub fn make_connector_response(capture: &storage::Capture) -> storage::ConnectorResponseNew {
+        storage::ConnectorResponseNew {
+            payment_id: capture.payment_id.clone(),
+            merchant_id: capture.merchant_id.clone(),
+            attempt_id: capture.capture_id.clone(), //capture_id will be connector_response's attempt_id
+            created_at: capture.created_at,
```

Comment:
these time field corresponds to Connector response table and these should be current time and not the capture's table time fields.